sweet/harnesses/cockroachdb.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2024 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package harnesses
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"golang.org/x/benchmarks/sweet/common"
+	"golang.org/x/benchmarks/sweet/common/log"
+)
+
+type CockroachDB struct{}
+
+func (h CockroachDB) CheckPrerequisites() error {
```

Comment:
[nit] I assume these methods are part of some interface, can we add comments mentioning that?

Reply:
Done.